sections/04_study.md

```diff
@@ -52,7 +52,104 @@ particularly notable in this area?*
+[@doi: 10.1109/TCBB.2014.2343960]. Zhou developed an iterative deep 
+learning framework to simultaneously predict secondary structure, 
+backbone torsion angles and solvent accessibility 
+[@doi:10.1038/srep11476]. However, none of these deep learning methods 
+achieved significant improvement over PSIPRED in terms of Q3 accuracy. In 
+2014, Zhou and Troyanskaya demonstrated that they could improve Q8 
+accuracy over a shallow learning architecture conditional neural fields [@doi: 
+10.1002/pmic.201100196] by using a deep supervised and convolutional 
+generative stochastic network[@arXiv:1403.1347], but did not report any 
+results in terms of Q3 accuracy. In 2016 Wang and Xu developed a deep 
+convolutional neural fields (DeepCNF) model and showed that this model 
+can significantly improve secondary structure prediction in terms of both Q3 
+and Q8 accuracy[@doi:10.1038/srep18962]. DeepCNF is the first that 
+reports Q3 accuracy of 84-85%, much higher than the 80% accuracy 
+maintained by PSIPRED for more than 10 years. It is also reported that 
+DeepCNF can improve prediction of solvent accessibility and disorder 
+regions [@doi:10.1007/978-3-319-46227-1_1].
+
+Contact-assisted protein folding represents a promising new direction for ab 
+initio folding of proteins without good templates in PDB, but it requires 
+accurate contact prediction. Evolutionary coupling analysis (ECA) is an 
+effective contact prediction method for some proteins with a very large 
+number (>1000) of sequence homologs, but ECA fares poorly for proteins 
+without many sequence homologs. Since (soluble) proteins with many 
+sequence homologs are very likely to have a good template in PDB, to make 
+contact-assisted folding practically useful for ab initio folding, it is essential 
+to predict accurate contacts for proteins without many sequence homologs. 
+By combining ECA with a few other protein features, shallow neural 
+network-based methods such as MetaPSICOV [@doi: 
+10.1093/bioinformatics/btu791] and CoinDCA-
+NN[@doi:10.1093/bioinformatics/btv472] have shown some advantage 
+over ECA for proteins with a small number of sequence homologs, but their 
+accuracy is still not very good. In recent years, deep learning methods have 
+been explored for contact prediction. For example, Di Lena et al. introduced 
+a deep spatio-temporal neural network (up to 100 layers) that utilizes both 
+spatial and temporal features to predict protein 
+contacts[@doi:10.1093/bioinformatics/bts475]. Cheng group combined 
+deep belief networks and boosting techniques to predict protein contacts 
+[@10.1093/bioinformatics/bts598] and trained deep networks by layer-wise 
+unsupervised learning followed by fine-tuning of the entire network. 
+Elofsson group developed an iterative deep learning technique for contact 
+prediction, in which Random Forests are applied to predict contacts at each 
+iteration [@doi:10.1371/journal.pcbi.1003889]. However, blindly tested in 
```

Comment:
They trained a RF to do prediction first, then feed the output of the first RF (and original input features) to the 2nd RF and then feed the output of the 2nd RF (and original input) to the 3rd RF. They repeated this 4-5 times. This is a typical deep learning method we talk about, but the authors thought their method is deep learning.